arto-converter/artoConverter.cpp

```diff
@@ -112,6 +112,85 @@ bool convert_arabic_to_roman(unsigned int arabic_number, char* roman_num)
   return true;
 }
 
+////////////////////////////////////////////////////////////////////////////////////
+//----------------------------------------------------------------------------------
+
+#define ILL_TERMINATOR  " ill"
+#define AMB_TERMINATOR  " amb"
+#define ERR_TERMINATOR  " err"
+#define AMB_DIGIT       -1
```

Comment:
```suggestion
#define UNDEF_DIGIT       -1
```
Предлагаю такое название